packages/react/src/lazy.ts

```diff
@@ -21,55 +104,18 @@ import { noop } from './utils/noop'
  * const Component3 = customLazy(() => import('./Component3'), { reload: 2 })
  * ```
  */
-const createLazy = (defaultOptions: { reload: number }) => {
+const createLazy = (defaultOptions: ReloadOptions) => {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const lazy = <T extends ComponentType<any>>(
     load: () => Promise<{ default: T }>,
-    options?: { reload: number }
+    options?: ReloadOptions
   ): LazyExoticComponent<T> & {
     load: () => Promise<void>
   } => {
-    const finalOptions = options || defaultOptions
-
-    if (process.env.NODE_ENV === 'development' && finalOptions.reload) {
-      if (!Number.isInteger(finalOptions.reload) || finalOptions.reload <= 0) {
-        console.error(
-          `[@suspensive/react] lazy: reload option must be a positive integer, but received ${finalOptions.reload}. ` +
-            `Please provide a positive integer value (e.g., { reload: 3 }).`
-        )
-      }
-    }
+    const finalOptions = parseOptions(options || defaultOptions)
```

Comment:
I'd like to follow TanStack's naming, it's really trivial things. I wanna share good refs

https://github.com/TanStack/query/blob/35d5d55aa72a9e39284554852983b0745deecc52/packages/react-query/src/useBaseQuery.ts#L55

```suggestion
    const defaultedOptions = getDefaultedOptions(options, defaultOptions)
```

I think we need to get defaulted options, not just parseOptions, because we can add more options. options could vary in near future